pyproject.toml

```diff
@@ -92,6 +92,7 @@ disable = [
   "unsupported-assignment-operation",
   "unsupported-membership-test",
   "unused-import",
+  "use-a-generator",
```

Comment:
Which line was throwing this?

Reply:
I'd originally written

```
if any(tz is not None for tz in timezones):
```
as a list comprehension. It just seemed like too minor of a thing to bother annoying other contributors with, but no objections to keeping this check enabled

Reply:
I would like to see this enabled. We had a different package check for this previously. 

The short circuiting behavior described in https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/use-a-generator.html seems like a useful check to guarantee

Reply:
Sure, have re-enabled

I might try to get this back into pyupgrade - it was taken out after some similar rewrites caused some performance degradations, but for `any` I'd imagine it would be a consistent improvement https://github.com/asottile/pyupgrade/issues/704

Reply:
Ah thanks for the context.